libs/template/ng-package.json

```diff
@@ -5,7 +5,7 @@
     "flatModuleFile": "template",
     "entryFile": "src/index.ts",
     "umdModuleIds": {
-      "@rx-angular/cdk": "rx-angular.cdk"
+      "@rx-angular/cdk": "@rx-angular/cdk"
```

Comment:
i remember @arturovt telling me that the prevous `dot notation` was actually correct. I need to test this out

Reply:
I don't get the point of using another name here

Reply:
I have tested the build in another project and it seems to work just fine without any warnings. I will do some more testing

Reply:
@hoebbelsB it will not show any warning during the build since UMD is a runtime:
1. you can check the `.umd.js` and it should show you the factory definition with `globals.umdId` here
2. try to create a simple project and add `@rx-angular/cdk` to webpack externals and try to import the UMD file through SystemJS for example (this is an example of sharing dependencies in micro-frontends)

As far as I can see in the latest version the UMD id is `rx-angular.cdk` (https://unpkg.com/@rx-angular/cdk@1.0.0-alpha.6/bundles/rx-angular-cdk.umd.js): `factory((global['rx-angular'] = global['rx-angular'] || {}, global['rx-angular'].cdk = {})`

Reply:
thx @arturovt

Reply:
I will bring this back, thanks, @arturovt can you check [this comment](https://github.com/rx-angular/rx-angular/pull/587#issuecomment-802868720), please?